lib/English.t

```diff
@@ -12,6 +15,8 @@ use English qw( -no_match_vars ) ;
 use Config;
 use Errno;
 
+no strict 'vars'; # Relax strictures because we're testing global variables
```

Comment:
The `English` vars should not require `no strict 'vars'`, if that's what you mean

Reply:
Yeah. I've turned up strict in the file and found a dead use of $threads from a previous removal of 5.005 threads.